karma.conf.js

```diff
@@ -1,128 +1,43 @@
-var argv = require('yargs').argv;
-var path = require('path');
-var webpack = require('webpack');
+import { testConfig } from './webpack.config';
+import { argv } from 'yargs';
+import * as path from 'path';
+import * as webpack from 'webpack';
 
-module.exports = function(config) {
+export default (config) => {
     config.set({
 
         // base path that will be used to resolve all patterns (eg. files, exclude)
-        basePath: __dirname,
+        // basePath: __dirname,
 
         // frameworks to use
         // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
         frameworks: [
-            'mocha',
-            'sinon-chai'
+            'mocha', 'chai'
         ],
 
         // list of files / patterns to load in the browser
         files: [
-            { pattern: 'node_modules/babel-polyfill/dist/polyfill.js', included: true, served: true },
-            { pattern: 'test/test.js', included: true, served: true }
+            
+            'test/index.js'
         ],
-
-        plugins: [
```

Comment:
Nope.  Polyfill is only needed if you need promises or something.  this lib is pretty small and doesn't need it.  test/test.js is now test/index.js.